src/main/resources/templates/footer.html

```diff
@@ -0,0 +1,13 @@
+<html lang="">
```

Comment:
dunno but i think i commented this before - are we sure we need the html and body tags?

Reply:
okay now I do understand